.github/pull_request_template.md

```diff
@@ -1,11 +1,12 @@
-# Resolves issue
+# Resolves HASH_SIGN_FOLLOWED_BY_ISSUE_NUMBER
```

Comment:
Just curious, is the word "issue" actually a problem? 

In this PR, all I did was to paste the link to my issue after the word "issue" here, and it looks like everything was linked correctly? https://github.com/microsoft/AzureTRE/pull/2359

Reply:
Although now I'm not sure, it looks like it might've been linked manually there, although don't think I did it

Reply:
I've seen multiple example where the issue wasn't picked up. always deleted the word issue and then it was ok...